src/commands/local.ts

```diff
@@ -610,6 +629,174 @@ function getBundler(classConfiguration: ClassConfiguration): BundlerInterface |
     return bundler;
 }
 
+async function runDockerContainer(databaseType: string, projectName: string, runCommand: string) {
```

Comment:
the name of the function suggests that it starts a general purpose docker, but what it actually does is that it starts a docker database. name it accordingly. one suggestion would be `startLocalDatabaseContainer` but you can also think of something else.